### PR TITLE
Added --skip-install parameter to backstage-cli versions:bump

### PR DESCRIPTION
**Pull request by wrighbr**
Signed-off-by: Brett Wright <brett.wright@nine.com.au>

## Hey, I just made a Pull Request!

Added the `--skip-install` parameter to `backstage-cli versions:bump`. 

This change allow you to skip the installation of the yarn packages when running the `backstage-cli versions:bump`

closes https://github.com/backstage/backstage/issues/15230

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ]  Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/witty-pears-explode.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': patch
+---
+
+Added --skip-install parameter to backstage-cli versions:bump
```
Comment (tudi2d) [inline .changeset/witty-pears-explode.md:5]: Optional: Improve readability of Changelog

```suggestion
Added `--skip-install` parameter to `backstage-cli versions:bump`
```

**Review by tudi2d — APPROVED** [review]
code looks good - let's get another ✅ from a maintainer

File: packages/cli/src/commands/versions/bump.ts [hunk 2]
```diff
@@ -255,7 +255,13 @@ export default async (opts: OptionValues) => {
       );
     }
 
-    await runYarnInstall();
+    if (opts.skipInstall === undefined) {
```
Comment (jhaals) [inline packages/cli/src/commands/versions/bump.ts:258]: Very minor thing but I think we could be simplified a little bit with
```suggestion
    if (!opts.skipInstall) {
```

Looks like that's what we do in [createApp](https://github.com/backstage/backstage/blob/master/packages/create-app/src/createApp.ts#L121)

**Comment by jhaals** [comment]
Nice work with this, just a minor thing and then let's :shipit:!

**Comment by wrighbr** [comment]
> Sorry @wrighbr I just realised that we don't have a test covering this change. Would you be able to add one to `bump.test.ts`, there should be examples in the tests for how to test that `yarn install` `.not.toHaveBeenCalled();` or similar

@jhaals I've created a test I'm pretty new to testing in javascript/typescript. It seems to work fine hopefully it will do.

**Review by jhaals — APPROVED** [review]
Thank you! :)

**Pull request merged**